text/string.go

```diff
@@ -9,11 +9,13 @@ import (
 
 // Constants
 const (
-	EscapeReset     = EscapeStart + "0" + EscapeStop
-	EscapeStart     = "\x1b["
-	EscapeStartRune = rune(27) // \x1b
-	EscapeStop      = "m"
-	EscapeStopRune  = 'm'
+	EscapeReset       = EscapeStart + "0" + EscapeStop
+	EscapeStart       = "\x1b["
+	EscapeStartAlt    = "\x1b]8"
+	EscapeStartRune   = rune(27) // \x1b
+	EscapeStop        = "m"
+	EscapeStopRune    = 'm'
+	EscapeStopRuneAlt = '\\'
```

Comment:
I wonder if there is a better way to do this for now and the future, in case another pattern needs to be supported. But this is a good start.